core/unit_test/CMakeLists.txt

```diff
@@ -844,6 +844,14 @@ if(Kokkos_ENABLE_HIP)
       UnitTestMain.cpp
       hip/TestHIP_InterOp_Streams.cpp
   )
+  if(KOKKOS_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 5.6)
```

Comment:
It seems usually the following is done:
```cmake
if(KOKKOS_CXX_COMPILER_ID STREQUAL "<name-of-hipcc-I-guess" AND KOKKOS_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 5.6)
```

Reply:
It would also help people reasoning about your condition and understand it is for `hipcc`.

Reply:
This is more aboutthe ROCm version than the compiler.

Reply:
nevermind, HIP macros aren't available. Working on a different solution.

Reply:
Used the suggestion to match other places using `KOKKOS_CXX_COMPILER_ID STREQUAL HIPCC`.